src/app/modules/rup/components/elementos/graficoLineal.component.ts

```diff
@@ -1,61 +1,68 @@
 import { Component, Output, Input, EventEmitter, OnInit } from '@angular/core';
-import { RUPComponent } from './../core/rup.component';
+import { RUPComponent } from '../core/rup.component';
 
 import * as moment from 'moment';
 import 'moment/locale/es';
+import { isNumber } from 'util';
 
 @Component({
-    selector: 'rup-seguimiento-del-peso',
-    templateUrl: 'seguimientoDelPeso.html'
+    selector: 'rup-grafico-lineal',
+    templateUrl: 'graficoLineal.html'
 })
-export class SeguimientoDelPesoComponent extends RUPComponent implements OnInit {
-    // variables para guardar los pesos de las prestaciones
-    public pesos: any[] = [];
+export class GraficoLinealComponent extends RUPComponent implements OnInit {
+    // variables para guardar los datosLineales de las prestaciones
+    public datosLineales: any[] = [];
 
     // opciones para el grafico
     public barChartOptions: any = {};
     public barChartLabels: any[];
     public barChartType = 'line';
     public barChartLegend = false;
     public barChartData: any[] = [];
+    barChartDates: any[] = [];
 
     ngOnInit() {
         moment.locale('es');
 
-        if (this.elementoRUP.conceptosBuscar && this.elementoRUP.conceptosBuscar.length > 0) {
 
-            // buscamos
-            this.prestacionesService.getRegistrosHuds(this.paciente.id, '<<27113001').subscribe(prestaciones => {
+        let datos = this.elementoRUP.params.map((param, index) => {
+
+            this.prestacionesService.getRegistrosHuds(this.paciente.id, `<<${param.conceptId}`).subscribe(prestaciones => {

```

Comment:
```suggestion
            this.prestacionesService.getRegistrosHuds(this.paciente.id, `${param.query}`).subscribe(prestaciones => {
```

@andrrrl  Como dice Silvina, debería hacer así, la query por parametro. Por si hay que pone cosas más complejas.

Reply:
Ah, si ese cambio ya estaba charlado, pero no quedó en el PR

Reply:
Ya quedó